docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +78,348 @@ To enable your application to use the image util functionality:
 
-   src = malloc(size);
-   ret = media_packet_get_buffer_data_ptr(src, &src_ptr);
-   ret = memcpy(src_ptr, img_source, size);
+   ```
+   ret = image_util_transform_destroy(transform_h);
    ```
 
-<a name="colorspace"></a>
-## Converting the Color Space
+<a name="mediapacket_colorspace"></a>
+## Converting Media Packet Color Space
 
-To convert an image from one color space to another:
+To convert an image from one color space of a media_packet to another:
 
-1. Create a transformation handle using the `image_util_transform_create()` function:
+1. Create a transformation handle using `image_util_transform_create()`:
 
    ```
-   transformation_h handle;
-   ret = image_util_transform_create(&handle);
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
-2. Set the target color space using the `image_util_transform_set_colorspace()` function:
+2. Set the target color space using `image_util_transform_set_colorspace()`:
 
    ```
-   ret = image_util_transform_set_colorspace(handle, colorspace);
+   ret = image_util_transform_set_colorspace(transform_h, colorspace);
    ```
 
-3. Execute the transformation using the `image_util_transform_run()` function:
+3. Execute the transformation using `image_util_transform_run()`:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_run(transform_h, (media_packet_h)src_packet,
+                                (image_util_transform_completed_cb)completed_callback,
+                                user_data);
    ```
 
    > **Note**
    >
-   > - Here, the `image_util_transform_run()` function only converts the color space. The function does not change the image width or height, or any other image property.
-   > - Because of the restrictions of the image processing library, not all color space combinations are supported for conversion. For example, the NV12 format is commonly used in hardware chips, but it is not supported by the library.
-   > - If hardware acceleration is enabled, you can execute 2 more image transformations using the same transformation handle.
+   > - `image_util_transform_run()` only converts the color space. The function does not change the image width, height, or any other image property. Due to these restrictions of the image processing library, not all color space combinations are supported for conversion.
+   > - For more information on how to use the media packet handle of the Media Tool API, see the [Media Handle Management](media-handle.md).
 
-4. Handle the transformation results in the `image_util_transform_completed_cb()` callback, which is invoked after the transformation is complete.
+5. Handle the transformation results in the `image_util_transform_completed_cb()` callback, which is invoked after the transformation is complete.
```

Comment:
Global: Suggest removing the and callback. Here in the `image_util_transform_completed_cb()`, cb denotes callback. Therefore, suggest rephrasing to:

Handle the transformation results in `image_util_transform_completed_cb()`, which is invoked after the transformation is complete.